### ISSUE TITLE
setHttpRequestOptions not available?

### ISSUE THREAD

**Issue by mrinkel** [zyra/ionic-image-loader#155]
I have ionic-image-loader working for retrieving images from by test server, but my production server requires a header value.  The documentation references the ability to add a header to the request: 

setHttpRequestOptions(options: any)

... but I am not seeing that option with ImageLoderConfig.

**Comment by mrinkel** [comment]
I worked around the issue by implementing a http interceptor as described by matheusdavidson in issue #145.  Works fine.

**Comment by cocowalla** [comment]
@madoBaker I don't see a `setHttpRequestOptions` method in the latest version, 6.3.2, or anywhere in the source code - has it been renamed to `setHttpHeaders`?

**Comment by madoBaker** [comment]
@cocowalla correct, will update the docs. Thanks for the heads up and it takes `HttpHeaders` from
